### DISCUSSION TITLE
XRDP clipboard feature

### DISCUSSION THREAD

**Discussion by devesh288** [neutrinolabs/xrdp#3142] [Q&A]
Hi Team,

I am trying to configure copy/paste feature from XRDP to client and vice versa for specific users but it is not working. Everytime it is picking global value defined in sesman.ini.

Need advise how to achieve the same as it is urgent.

Hope will get some response.

**Comment by matt335672** [comment]
@devesh288 - if I'm understanding your question, we don't support per-user values for this feature.

**Reply by devesh288** [reply]
Thanks Matt.

Want to know is there any workaround available to override sesman.ini values with my custom value not for specific user but globally.

**Reply by matt335672** [reply]
You can specify a different sesman.ini file with `-c`, but that's it. What do you mean by 'global' exactly? Isn't that the same as sesman.ini?

**Reply by devesh288** [reply]
Example:

in actual sesman.ini file having allowOutboundRestriction=all which is use as global parameter all XRDP sessions..
I want to use another sesman.ini file and assign to specific user session so user session using global value use custom values

**Reply by matt335672** [reply]
We have no way to do that I'm afraid.

We'd need a way to map specific user sessions to configurations, and we don't have anything like that.
